extract_text_from_pptx.py

```diff
@@ -0,0 +1,30 @@
+from pptx import Presentation
+
+
+def extract_text_from_pptx(file_path):
+    """
+        Extracts text from a .pptx file using the python-pptx package.
+
+        Args:
+            file_path (str): The path to the .pptx file.
+
+        Returns:
+            str: The extracted text from the .pptx file.
+    """
+
+    presentation = Presentation(file_path)
+    extracted_text = ""
+    for slide in presentation.slides:
+        for shape in slide.shapes:
+            if shape.has_text_frame:
+                for paragraph in shape.text_frame.paragraphs:
+                    for run in paragraph.runs:
+                        extracted_text += run.text + " "
+            elif shape.has_table:
+                table = shape.table
+                for row in table.rows:
+                    for cell in row.cells:
+                        for paragraph in cell.text_frame.paragraphs:
+                            for run in paragraph.runs:
+                                extracted_text += run.text + " "
```

Comment:
Try to make this code cleaner

1. Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

2. Replace for loops -  
https://medium.com/python-pandemonium/never-write-for-loops-again-91a5a4c84baf
Replace for loops with more advanced python features 

3.  Replace these two for loops with a list comprehension with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python

Reply:
Fixed it